src/Worker/UpdateServerPeers.php

```diff
@@ -57,13 +57,13 @@ public static function execute(string $url)
 		$total = 0;
 		$added = 0;
 		foreach ($peers as $peer) {
-			if (Network::isUrlBlocked('http://' . $peer)) {
+			if (Network::isUrlBlocked('https://' . $peer)) {
 				// Ignore blocked systems as soon as possible in the loop to avoid being slowed down by tar pits
 				continue;
 			}
 
 			++$total;
-			if (DBA::exists('gserver', ['nurl' => Strings::normaliseLink('http://' . $peer)])) {
+			if (DBA::exists('gserver', ['nurl' => 'http://' . $peer])) {
```

Comment:
On normalizing we also remove trailing `/` and remove leading `www.`. This replacement does only half of the job.

Reply:
Please tell this @MrPetovan and not me. I never touched that code before, just added the `if()` blocks for checking if the URL is blocked. We should use the original URL for checking against block list. What if someone wants to block the `www.` part of an URL? Some have `www.friendica.tld` as their "homepage".